### PR TITLE
Capabilities: Events

### PR DESCRIPTION
**Pull request by NickFitz**
Expanded on the event subscription capability.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5485,13 +5532,62 @@ <h5>Conclusion</h5>
...
+Responding to changes in the state of a map, or user interactions with elements of the map such as markers,
+is necessary for a web application to usefully modify its own state so as to reflect the results of such interactions.
 </p>
 <ul data-ucr-role="conclusion-notes">
 <!-- bullet-point notes summarizing implementation details / blocking issues.
 Add tags to the start of a point with an empty `<a data-ucr-role="tag"></a>` where the text content or href matches a tag `id` value. -->
-<li><a data-ucr-role="tag" href=""></a>
-<!-- short summary of how the tag applies -->
+<li><a data-ucr-role="tag">privacy-fingerprinting</a>
+As scripts may be loaded from arbitrary origins (depending on cross-rigin restrictions),
```
Comment (Malvoz) [inline index.html:5543]: Typo "cross-rigin"

Comment (NickFitz) [reply index.html]: Oops! Fixed, thanks :-)

…though I've just realised WebStorm has somehow switched itself to my other GitHub account for that commit.

File: index.html [hunk 2]
```diff
@@ -5485,13 +5532,62 @@ <h5>Conclusion</h5>
...
 <ul data-ucr-role="conclusion-notes">
 <!-- bullet-point notes summarizing implementation details / blocking issues.
 Add tags to the start of a point with an empty `<a data-ucr-role="tag"></a>` where the text content or href matches a tag `id` value. -->
-<li><a data-ucr-role="tag" href=""></a>
-<!-- short summary of how the tag applies -->
+<li><a data-ucr-role="tag">privacy-fingerprinting</a>
+As scripts may be loaded from arbitrary origins (depending on cross-origin restrictions),
+an event object including information about a map might allow deductions to be made about a viewer's device.
+For example, the bounds of a map in a responsive design could vary depending on the device on which it is viewed,
+allowing a third-party script to determine that a user is on a handheld device such as a phone.
```
Comment (AmeliaBR) [inline index.html:5546]: Glad to see you thinking about these details, but this isn't really new fingerprinting surface: scripts can access device size directly.

**Comment by NickFitz** [comment]
> Hi Nick,
> 
> This is a great write-up. I think you've got the level of detail correct, no need to split it up into smaller capabilities.
> 
> Don't go overboard with the tags on the conclusions, though. Some of these are really stretching it, and a long list that's the same for every requirement will have less impact that a few key points that are unique and relevant.
> 
> Let me know if you want to edit the tags yourself, otherwise I will probably merge and then clean up in a separate pass.

Yes, on reflection I did get a bit carried away with the tags there. I've removed some, but feel free to cull any more of them as needed.

File: index.html [hunk 3]
```diff
@@ -4543,24 +4540,57 @@ <h5>Existing implementations</h5>
...
 <dt>leaflet-api</dt>
 <dt>open-layers-api</dt>
+<dt>tomtom</dt>
 <dd><a>full support</a>:
-These APIs all define requirements for a JavaScript object or DOM element
-to be able to be used as a custom control.
+<p>
+All of these implementations fully support the concept of controls,
+including support for control implementations to be notifed when they are added to or removed from the map,
+so they can update their own internal state acordingly.
```
Comment (Malvoz) [inline index.html:4551]: s/acordingly/accordingly

File: index.html [hunk 4]
```diff
@@ -4585,7 +4617,10 @@ <h5>Conclusion</h5>
 See examples.
 -->
 <p>
-Based on the limited data, we are <a data-ucr-role="conclusion">undecided</a> about whether this should be a requirement.
+It might seem that, as controls are implemented using existing interactive HTML elements,
+there is no pressing need for them to be supported by additional APIS.
```
Comment (Malvoz) [inline index.html:4621]: s/APIS/APIs

File: index.html (hunk 3 above)
Comment (NickFitz) [reply index.html:4551]: And "notifed" in the line above :-)

File: index.html (hunk 4 above)
Comment (NickFitz) [reply index.html]: I've bumped up the highlighting of typos in my IDE in the hope I catch more of these before committing! Thanks for spotting them :-)

**Comment by AmeliaBR** [comment]
Thanks for the edits, Nick, and for the copy-edits, Robert. Merging this now — any further additions can be made separately.

**Pull request merged**